client/blocks/data-center-picker/index.tsx

```diff
@@ -88,7 +80,14 @@ const DataCenterPicker = ( {
 	value,
 }: Props ) => {
 	const [ isFormShowing, setIsFormShowing ] = useState( false );
-	const { hasTranslation } = useI18n();
+	const hasEnTranslation = useHasEnTranslation();
```

Comment:
I realized I should use `hasEnTranslation` because it adds a check for EN locale. If EN locale or if the text is translated, show the text. That way it shows up in English, which doesn't need to wait for translations.